README.md

```diff
@@ -27,14 +27,14 @@ These videos on TikTok, YouTube and Instagram get MILLIONS of views across all p
 
 - Python 3.6+
 - Playwright (this should install automatically during installation)
-
+- pip
```

Comment:
Pip would have been installed when the user installed python, and doesn't need to be listed here